### PR TITLE
Update README with installation and build instructions.

### PR DESCRIPTION
**Pull request by viklund**
Are these installation instructions ok, or should there be something more?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,21 @@ SFTP-Squid - CLI for transferring files between sftp servers
 
 Simple usage:
 
-    $ ./sftpsquid user1@host1:file user2@host2:
```
Comment (jhagberg) [inline README.md:6]: can you motivate this change?

File: README.md [hunk 2]
```diff
@@ -3,7 +3,21 @@ SFTP-Squid - CLI for transferring files between sftp servers
...
+
+Installation instructions
+-------------------------
+
+### Linux and MacOS:
+
+1. Download the latest release from [github](https://github.com/BILS/sftp-squid/releases)
+2. Unzip the file.
+
+Then you can run it either from inside the directory or you can move the two
```
Comment (jhagberg) [inline README.md:17]: okey this explains it.

File: README.md [hunk 3]
```diff
@@ -54,6 +73,6 @@ factor password is set as well which is `hej`.
 
 ### Test
 
-To then test the transfer of one file do this:
+To then test the transfer of one directory do this:
 
     $ ./sftpsquid test_user@localhost:4021:transfer test_user@localhost:4022:
```
Comment (jhagberg) [inline README.md]: should this also be without "./" ?

Comment (viklund) [inline README.md]: Yes, fix comming.

**Comment by jhagberg** [comment]
looks ok and easy!

**Pull request merged**